test/testcase.jl

```diff
@@ -68,7 +92,22 @@ function testcase(
             data    = CSV.File(joinpath(postprodir, file); header=1) |> DataFrame
             data    = data[1:size(dataref, 1), :]
 
-            test = isapprox.(data, dataref; rtol=rtol, atol=atol)
+            # Check the number of columns matches, before removing any excluded_columns
+            @test ncol(data) == ncol(dataref)
+            for col ∈ excluded_columns
+                select!(data, Not(Cols(contains(col))))
+                select!(dataref, Not(Cols(contains(col))))
+            end
+
+            fdata = asfloat.(data)
```

Comment:
The `error-indicators.csv` file writes a final row with the mean value. This quantity is what is actually being used to drive the adaptation, and combines all the proceeding rows. Rather than apply a special meaning to maybe `-1` or the like, I opted for making machine parsing the csv more complicated as a human reading it would find it much more intuitive.

Reply:
I do not think we should make changes which change the data type of the output CSV files. To me the simplest solution is to remove this row, since it's easily computable from the other data in the file.

Reply:
done https://github.com/awslabs/palace/pull/85/commits/fc5000601a6bd4ede4cfe331884a67c37bcb7054